src/binary-search-tree.js

```diff
@@ -1,10 +1,68 @@
-// https://msdn.microsoft.com/en-us/library/aa289150(v=vs.71).aspx
+
+class Queue {
```

Comment:
You can include your queue implementation in another file and then import it into this file so that you don't need to have the queue implementation in the same file as well.